.github/workflows/test.yml

```diff
@@ -10,7 +10,13 @@ jobs:
   build: # make sure build/ci work properly
     runs-on: ubuntu-latest
     steps:
```

Comment:
Tu dis dans la description que tu as fait:
- Migration de set-output vers la variable [$GITHUB_OUTPUT](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/)

Mais je ne vois aucun changement par rapport à cela 🤔 Est-ce que je manque quelque chose?